neofetch

```diff
@@ -993,6 +993,12 @@ get_distro() {
                     *) distro="Siduction ($(lsb_release -sic))"
                 esac
 
+            elif [[ -f /etc/skiffos-release ]]; then
+                case $distro_shorthand in
+                    on|tiny) distro=SkiffOS ;;
+                    *) distro="SkiffOS ($(cat /etc/skiff-release | grep PRETTY_NAME | awk -F'"' '$0=$2'))"
+                esac
+
```

Comment:
Existing detection methods don't work?

Reply:
/etc/os-release:

```
NAME=SkiffOS
VERSION=2021.02.2-31-g9badfd5
ID=skiff
VERSION_ID=9badfd5
PRETTY_NAME="SkiffOS 2021.02.2-31-g9badfd5"
BUILD_DATE="Thu Jun 10 17:11:44 PDT 2021"
BUILD_USER="paralin@forge"
VERSION_FULL="SkiffOS version 2021.02.2-31-g9badfd5 (paralin@forge) Thu Jun 10 17:11:44 PDT 2021"
```

/etc/skiff-release

```
NAME=SkiffOS
VERSION=2021.02.2-31-g9badfd5
ID=skiff
VERSION_ID=9badfd5
PRETTY_NAME="SkiffOS 2021.02.2-31-g9badfd5"
BUILD_DATE="Thu Jun 10 17:11:44 PDT 2021"
BUILD_USER="paralin@forge"
VERSION_FULL="SkiffOS version 2021.02.2-31-g9badfd5 (paralin@forge) Thu Jun 10 17:11:44 PDT 2021"
BUILD_CONFIG="pine64/common,skiff/moduleimg,pine64/phone,apps/docker,skiff/core,core/gentoo"
```

Reply:
The current `master` fails on my end, I bisected it to commit `41f75b7b484cca653e0db19b194d99a993dbf5f8`

neofetch: line 6364: syntax error near unexpected token `)'
neofetch: line 6364: `        "CelOS"*)'

Reply:
https://github.com/dylanaraps/neofetch/pull/1772 fixed here

Reply:
OK @dylanaraps I simplified it to use existing os-release detection, works fine.